projects/1.20/assets/braziliandelight/braziliandelight/lang/zh_cn.json

```
@@ -0,0 +1,278 @@
+{
+  "advancements.braziliandelight.acai_berry_item.description": "获得巴西莓",
+  "advancements.braziliandelight.acai_berry_item.title": "巴西莓乐事",
+  "advancements.braziliandelight.acai_cream_craft.description": "制作巴西莓奶油球",
+  "advancements.braziliandelight.acai_cream_craft.title": "热带和睦",
+  "advancements.braziliandelight.beans_crop.description": "种植和收获豆荚",
+  "advancements.braziliandelight.beans_crop.title": "感豆身授！",
+  "advancements.braziliandelight.brigadeiro_craft.description": "制作巴西巧克力球",
+  "advancements.braziliandelight.brigadeiro_craft.title": "巧克力珍宝",
+  "advancements.braziliandelight.cassava_crop.description": "种植和收获木薯",
+  "advancements.braziliandelight.cassava_crop.title": "复苏的根茎！",
+  "advancements.braziliandelight.cassava_fritters_craft.description": "制作炸木薯酪饼",
+  "advancements.braziliandelight.cassava_fritters_craft.title": "可口的根茎",
+  "advancements.braziliandelight.cheese_bread_craft.description": "制作奶酪面包",
+  "advancements.braziliandelight.cheese_bread_craft.title": "哇塞！",
+  "advancements.braziliandelight.cheese_making.description": "在锅里把细盐、柠檬和重奶油混合在一起制作奶酪！",
+  "advancements.braziliandelight.cheese_making.title": "手工奶酪",
+  "advancements.braziliandelight.chimarrao_craft.description": "制作马黛茶",
+  "advancements.braziliandelight.chimarrao_craft.title": "高乔烹茶",
+  "advancements.braziliandelight.coconut_cream_craft.description": "制作椰子奶油球",
+  "advancements.braziliandelight.coconut_cream_craft.title": "奶油味椰子",
+  "advancements.braziliandelight.coconut_drink_craft.description": "制作椰饮",
+  "advancements.braziliandelight.coconut_drink_craft.title": "桑巴啜饮",
+  "advancements.braziliandelight.coconut_item.description": "获得椰子",
+  "advancements.braziliandelight.coconut_item.title": "坚如磐石！",
+  "advancements.braziliandelight.coffee_crop.description": "种植和收获咖啡浆果",
+  "advancements.braziliandelight.coffee_crop.title": "冲泡觉醒！",
+  "advancements.braziliandelight.collard_greens_crop.description": "种植和收获羽衣甘蓝",
+  "advancements.braziliandelight.collard_greens_crop.title": "绿色环保！",
+  "advancements.braziliandelight.condensed_milk_craft.description": "制作炼乳",
+  "advancements.braziliandelight.condensed_milk_craft.title": "甜蜜精华",
+  "advancements.braziliandelight.corn_crop.description": "种植和收获玉米",
+  "advancements.braziliandelight.corn_crop.title": "爆谷！",
+  "advancements.braziliandelight.couscous_craft.description": "制作古斯米",
+  "advancements.braziliandelight.couscous_craft.title": "古斯米，蒸装待发！",
+  "advancements.braziliandelight.coxinha_craft.description": "制作炸鸡肉包",
+  "advancements.braziliandelight.coxinha_craft.title": "那咸咸的玩意是什么来的？",
+  "advancements.braziliandelight.feijoada_craft.description": "制作腓秀雅杂",
+  "advancements.braziliandelight.feijoada_craft.title": "腓秀雅杂盛宴",
+  "advancements.braziliandelight.fish_moqueca_craft.description": "制作莫奎卡焖鲙",
```

```suggestion
  "advancements.braziliandelight.fish_moqueca_craft.description": "制作巴西海鲜炖",
```
`巴西海鲜炖`、`莫奎卡炖鱼`、`莫奎卡海鲜炖`、`巴西炖鱼`等排列组合均可
`焖鲙`不行，`鲙`不通`烩`，也不是焖的